prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -14,35 +14,68 @@ identifier_1:
   constants:
     sys: "urn:id:extID"
 
-identifier_2:
+identifier_2a:
```

Comment:
Breaking this out into separate entries here to match up more closely with the ServiceRequest breakdown, also because OBR.2/3 should only be used if ORC.2/3 are null. Probably in practice will be functionally equivalent either way, if it's preferable to revert this and change ServiceRequest accordingly, I can do that too.